src/test/java/com/commercetools/sync/categories/CategorySyncMockUtils.java

```diff
@@ -137,35 +138,13 @@ public static CategoryDraft getMockCategoryDraft(@Nonnull final Locale locale,
         return categoryDraft;
     }
 
-    /**
-     * Given a {@code locale}, {@code name}, {@code slug}, {@code key}, {@code description},
-     * {@code metaDescription}, {@code metaTitle}, {@code metaKeywords}, {@code orderHint} and
-     * {@code parentId}; this method creates a mock of {@link CategoryDraft} with all those supplied fields. All the
-     * supplied arguments are given as {@link String} and the method internally converts them to their required types.
-     * For example, for all the fields that require a {@link LocalizedString} as a value type; the method creates an
-     * instance of a {@link LocalizedString} with the given {@link String} and {@link Locale}.
-     *
-     * @param locale          the locale to create with all the {@link LocalizedString} instances.
-     * @param name            the name of the category.
-     * @param key             the key id of the category.
-     * @param parentId        the id of the parent category.
-     * @param customTypeId    the id of the custom type of category.
-     * @param customFields    the custom fields of the category.
-     * @return an instance {@link CategoryDraft} with all the given fields set in the given {@link Locale}.
-     */
     public static CategoryDraft getMockCategoryDraft(@Nonnull final Locale locale,
```

Comment:
@andrii-kovalenko-ct but why remove the Javadoc :)?

Reply:
it is not removed, it has been moved 2 methods below:
https://github.com/commercetools/commercetools-sync-java/pull/154/files/f2adc9546d81b956be020df546fcfe997eca7202..221aa0ee8ec160c96cc2a8bfa895bcdb837ed125#diff-22c0470e28c10be39f1f22c58094eb65R177